main.js

```diff
@@ -0,0 +1,74 @@
+var letsCookButton = document.querySelector('.cook-button');
+var cookpotImage = document.querySelector('.cookpot');
+var dishPrinted = document.querySelector('.dish-output');
+var dishHeading = document.querySelector('.dish-heading');
+var clearButton = document.querySelector('.clear');
+
+// var currentSide = sides[getRandomIndex(sides)];
```

Comment:
Need to clarify best practice about declaring these here, vs just using them in the function where needed.